src/codegen_function.py

```diff
@@ -1077,9 +1110,20 @@ def main():
     arg_parser.add_argument("-v", "--verbosity-level", type=int, default=LOG_INFO,
                             help="How verbose is the output. < 0 = silent, "
                                  "0 = only errors, 1 = normal, 2 = verbose, > 3 = debug")
+    arg_parser.add_argument('--nargs', help='Input String', nargs='*', type=str)
```

Comment:
What are we using nargs for? The C-preprocessor? If that's the case, can we call it `c-preprocessor-args`? If not, what is this for?

Reply:
the problem is that the `c-preprocessor-args` passed tough the cmake are seen as args of the command. renamed